calcom/cal.com
# Fix language of locations on booking page

## What does this PR do?

Fixes that locations on the booking page are always shown in English.

Fixes #2584 

Before: 
<img width="822" alt="Screenshot 2022-06-03 at 22 48 47" src="https://user-images.githubusercontent.com/30310907/172121784-f68ea2b1-7828-4944-bc9e-caa54a2b95dc.png">

After: 
<img width="832" alt="Screenshot 2022-06-03 at 22 46 15" src="https://user-images.githubusercontent.com/30310907/172121809-b8ed3819-61fe-4955-93ed-8cf7daca7af9.png">

**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?

- Change browser language to something else than english
- Go to booking page of an event type that has several locations
- See that locations are shown in your set language

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
